packages/ansible.rb

```diff
@@ -3,29 +3,20 @@
 class Ansible < Package
   description 'Ansible is a radically simple IT automation engine that automates cloud provisioning, configuration management, application deployment, intra-service orchestration, and many other IT needs.'
   homepage 'https://www.ansible.com/'
-  version '2.9.14'
+  version '3.3.0'
   license 'GPL-3'
   compatibility 'all'
-  source_url 'https://releases.ansible.com/ansible/ansible-2.9.14.tar.gz'
-  source_sha256 'efe7183e08f0c9dd9efd9124efa8978be62851b1e8eb856593a073538bfc6fb5'
+  source_url 'https://files.pythonhosted.org/packages/53/d5/6cdea3c8479644b1b788d57cad500e9d782e06f6a84cac05121a65cca67e/ansible-3.3.0.tar.gz'
```

Comment:
Why are we using ansible 2.x for ansible_base and 3.x here?

Reply:
look at the versioning on https://pypi.org/project/ansible and https://pypi.org/project/ansible-base

Reply:
I looked at it already and noticed the versioning was different.  This is not convincing me there is much reason to have separate packages.